astroquery/utils/commons.py

```diff
@@ -51,8 +51,10 @@ def FK4CoordGenerator(*args, **kwargs):
            'TableList',
            'suppress_vo_warnings',
            'validate_email',
+           'ASTROPY_LT_4_0',
            'ASTROPY_LT_4_1']
 
+ASTROPY_LT_4_0 = not minversion('astropy', '4.0')
```

Comment:
Is this suppose to work with 4.0, or just 4.0.2?

Reply:
Yes, if we stop touching the hidden stuff, `sources` is available since 4.0. That was added in an earlier PR.

Reply:
oh, that's perfect! thanks